### PR TITLE
Style index page

### PR DESCRIPTION
**Pull request by eraulo**
- New style to index page

### CODE REVIEW

**Comment by eraulo** [comment]
Kolega sira ne'e style foun ba home page, se kolega sira aseita ita bele usa ne'e, se lakohi ita bele nafatin ida uluk ne'e. ida ne'e rai tiha ba draf.Obrigado

**Comment by MarioAP** [comment]
Sujere uza nafatin ida uluk, ida nee hare cancel tia, kontinue halo furak tan style ida tuan nee

File: map/templates/map/fullmapview.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <script type="text/javascript" src="{% static 'js/bootstrap.min.js' %}"></script>
+        <script type="text/javascript" src="{% static 'js/leaflet.js' %}"></script>
+        <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap.min.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/timor.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/leaflet.css' %}">
+    </head>
+    <body>
+        <style>
+            #mapid {
+                height: 550px!important;
```
Comment (joanico) [inline map/templates/map/fullmapview.html:15]: La presiza uza important iha ne'e.

File: map/templates/map/details.html [hunk 2]
```diff
@@ -3,34 +3,37 @@
 {% block title %}{% trans 'Detail Timor GIS' %}{% endblock %}
 {% block content %}
     <style>
```
Comment (joanico) [inline map/templates/map/details.html:5]: @eraulo se bele muda hotu style sira husi template ba hotu css static file atu nune bele hanesan.

File: map/templates/map/mapview.html [hunk 3]
```diff
@@ -2,37 +2,55 @@
 {% load static i18n %}
 {% block title %}Timor Journey{% endblock %}
 {% block content %}
-    <div id="mapid"></div>
-    <div class="container">
-        <div class="row navbar-journey">
-            <div class="col-md-8">
-                <h3 class="trip-text">{% trans 'Our histories for our trip' %}</h3>
+<style>
```
Comment (joanico) [inline map/templates/map/mapview.html:5]: Bele muda style ba css file

**Comment by joanico** [comment]
@eraulo se bele hasai user no deskrisaun viajen,  no fo sai deit foto 3 kada viajen sira seluk bele kria iha "More Details"

File: map/templates/map/details.html (hunk 2 above)
Comment (MarioAP) [reply map/templates/map/details.html:5]: Ano ita bele lao tuir ona style ida nee, maibe ita bele muda C

**Comment by PeteCoward** [comment]
Need to resolve the conflicts before further action

**Comment by PeteCoward** [comment]
to be replaced by #156

**Review by joanico — COMMENTED** [review]
@eraulo could you change the `.container` to `.container-fluid` to make full-width container in the base template, because map and content look very small.

**Comment by marobo** [comment]
@eraulo , I suggest to close this PR and then reopen it, because there is had a lot of conflicts.
If you are trying to solve this conflict maybe be hard.

**Pull request closed without merging**